lambdanotes/src/components/NoteList.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
+import './NoteList.css';
+import {connect} from 'react-redux';
+import {loadNotes} from '../actions';
```

Comment:
spaces around `{ connect }` and `{ loadNotes }` 
Nitpicky, I know but it follows convention